syncserver/src/web/handlers.rs

```diff
@@ -32,6 +33,15 @@ pub async fn get_collections(
     db_pool: DbTransactionPool,
     request: HttpRequest,
 ) -> Result<HttpResponse, ApiError> {
+    let _hashed_fxa_uid = String::from(&meta.user_id.hashed_fxa_uid);
+    let _user_agent = request
+        .headers()
+        .get("USER_AGENT")
+        .map(|header| header.to_str().unwrap_or("none"))
```

Comment:
I'll always argue `and_then` w/ `ok` is worth doing in cases like these so you don't have to duplicate the final `unwrap_or` case
```suggestion
        .and_then(|header| header.to_str().ok())
```

Reply:
Funny enough I talked to @linabutler abut this and she suggested the same, better pattern for sure.